SLACK-GUIDELINES.md

```diff
@@ -128,6 +128,16 @@ in the traditional sense.
 In the case that certain channels have rules or guidelines, they will be listed
 in the purpose or pinned docs of that channel.
 
+## Threaded Conversations
+We believe Slack threads in public channels are extremely useful for quickly and openly collaborating with your fellow contributors.  
+
```

Comment:
```suggestion

Slack threads in public channels are extremely useful for quickly and openly collaborating with your fellow contributors.  
```

"we believe" doesn't really fit the tone of the rest of the document, and I think the sentence works equally well without it.